docs/TEST_E2E.md

```diff
@@ -17,3 +17,27 @@ With this approach, the follow things can be tested:
 -   Whether or not certain features show/not show depending on the status of the users auth
 -   Run requests directly against the backend and see if we get results back
 -   Clicking any follow up buttons (including examples)
+
+## Flare Chat E2E Test flow (Not implemented yet)
```

Comment:
Didn't know where this one should live. The old approach has nothing to do with language servers but the new one will so technically this could live in here or lsp.md

Reply:
one doc for test info is useful. I wonder why TEST_E2E is not part of TESTPLAN (which could be renamed)